tests/kafkatest/services/kafka/kafka.py

```diff
@@ -191,7 +192,7 @@ def prop_file(self, node):
         # TODO - clean up duplicate configuration logic
         prop_file = cfg.render()
         prop_file += self.render('kafka.properties', node=node, broker_id=self.idx(node),
-                                 security_config=self.security_config)
+                                 security_config=self.security_config, num_nodes=self.num_nodes)
```

Comment:
Hmm, is this required? Normally `render` should automatically have access to all fields of the object it's being called on. I'm not sure why the `security_config` one is there either.